Fix: API 호출 query 부분 오류 수정

eunji-0623 opened the pull request:
- API 호출 시 query 관련 parameter가 제대로 전달되지 않는 오류를 수정했습니다.
- 나의 대시보드 페이지 > 초대받은 대시보드 부분에 모바일 디자인 추가했습니다.

eunji-0623 committed:
Fix: API 호출 query 부분 오류 수정

foxholic9 reviewed (approved):
확인했습니다. params 잘 사용해주신 것 같습니다.

eunji-0623 committed:
Feat: DashboardList에 mockData -> API 호출로 변경

- mockData를 사용하던 부분을 API 호출로 변경했습니다.
- API 호출로 변경함에 따라 usePagination도 일부 수정했습니다.

eunji-0623 committed:
Feat: Pagination 버튼에 disabled 처리 추가

eunji-0623 committed:
Fix: type error 수정

eunji-0623 committed:
Fix: linter 오류 수정

eunji-0623 committed:
Style: Table 스타일 수정

eunji-0623 committed:
Fix: 로그인 API 타입 오류 수정

eunji-0623 committed:
Feat: 초대받은 대시보드 모바일 반응형 디자인 추가

ghost commented:
확인했습니다!
